### ISSUE TITLE
windows use sanic not module syslog

### ISSUE THREAD

**Issue by zeromake** [channelcat/sanic#1027]
### env
1. system: windows7-64bit
2. python: python3.6.3-64bit
3. sanic: 0.6.0
### use
main.py
``` python
from sanic import Sanic
from sanic.response import json

app = Sanic(__name__)

@app.route("/")
async def test(request):
    return json({"message": 'ok'})
if __name__ == '__main__':
    app.run(host='0.0.0.0', port=8000)
```
python on windows is not syslog module
``` shell
File "C:\Users\zeromake\.virtualenvs\zero-reader-gYPzEBV9\lib\site-packages\sanic\config.py", line 4, in <module>
    import syslog
ModuleNotFoundError: No module named 'syslog'
```
diff config.bak.py config.py
``` diff
3,7c3
< try:
<     import syslog
< except ImportError:
<     class syslog():
<         LOG_DAEMON = 24
---
> import syslog
```
`python main.py` is not problem
``` shell
2017-11-18 22:46:48 - (sanic)[ERROR]: Traceback (most recent call last):
  File "C:\Users\zeromake\.virtualenvs\zero-reader-gYPzEBV9\lib\site-packages\sanic\server.py", line 143, in connection_timeout
    raise RequestTimeout('Request Timeout')
sanic.exceptions.RequestTimeout: Request Timeout

ERROR:sanic:Traceback (most recent call last):
  File "C:\Users\zeromake\.virtualenvs\zero-reader-gYPzEBV9\lib\site-packages\sanic\server.py", line 143, in connection_timeout
    raise RequestTimeout('Request Timeout')
sanic.exceptions.RequestTimeout: Request Timeout
```
but RequestTimeout is what happened，it happen on browser request end after some time

**Comment by m43k** [comment]
Looks like a duplicate of the "feature" reported in #856 
Have a look at the workarounds described there as well as the there referenced [https://stackoverflow.com/questions/25537419/python-import-syslog-on-windows](url)

**Comment by skewty** [comment]
Fixed in 21435c1

**Comment by seemethere** [comment]
Closing per @skewty's comment
